README.md

```diff
@@ -37,6 +37,8 @@ npm i use-debounce --save
 
 ### use-debounce
 
+Introduction -A Debouncing Events in ReactJS will allow you to call a function that ensures that a time-consuming task does not fire so often.
```

Comment:
debounce is used not only for task consuming operations. But also, to reduce noisiness for people.